frontend/src/components/SidebarRight.jsx

```diff
@@ -35,7 +35,7 @@ const useSidebarRight = () => {
             .catch((err) => {
                 console.log(err)
             })
-    }, [])
+    })
```

Comment:
Please provide dependency for variables used in useEffect. If the variable is a just a const and doesn't depend on each render, you can define it inside the useEffect (for example the baseUrl case).